lib/plans/checkout.dart

```diff
@@ -409,7 +406,7 @@ class _CheckoutState extends State<Checkout>
                           text: 'continue'.i18n,
                           // for Pro users renewing their accounts, we always have an e-mail address
                           // so it's unnecessary to disable the continue button
-                          disabled: !widget.isPro ? !showContinueButton : false,
+                          disabled: !enableContinueButton(),
```

Comment:
@atavism  I was able to click the continue button without email so, I changed to use !enableContinueButton()